web/src/scroll_util.ts

```diff
@@ -77,7 +77,11 @@ export function scroll_element_into_container(
     // this will be non-intrusive to users when they already have
     // the element visible.
     $container = get_scroll_element($container);
-    const elem_top = $elem.position().top - sticky_header_height;
+
+    const elem_offset = $elem.offset()?.top ?? 0;
+    const container_offset = $container.offset()?.top ?? 0;
+
+    const elem_top = elem_offset - container_offset - sticky_header_height;
```

Comment:
Hmm, can you explain why `.offset` is correct for the position of something that might be offscreen in a scroll container? This logic at least seems like it can't possibly be right if we're inside a scrollable area within another scrollable area, which we do have in a few places in the app.

I kinda feel like what we need is the offset within the relevant scroll container, not the offset to the document. Oh, that's what you're doing with `container_offset`. OK I can write a comment that makes this easier to undestand.

Reply:
``` diff
diff --git a/web/src/scroll_util.ts b/web/src/scroll_util.ts
index 6f4770950e..827d74cd0c 100644
--- a/web/src/scroll_util.ts
+++ b/web/src/scroll_util.ts
@@ -78,6 +78,9 @@ export function scroll_element_into_container(
     // the element visible.
     $container = get_scroll_element($container);
 
+    // To correctly compute the offset of the element's scroll
+    // position within our scroll container, we need to substract the
+    // scroll container's own offset within the document.
     const elem_offset = $elem.offset()?.top ?? 0;
     const container_offset = $container.offset()?.top ?? 0;
 
```